rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -594,7 +594,7 @@ error_code cellGameDataCheck(u32 type, vm::cptr<char> dirName, vm::ptr<CellGameC
 	{
 		switch (type)
 		{
-		case CELL_GAME_GAMETYPE_HDD: return cat != "HG"sv && cat != "AM"sv && cat != "AP"sv && cat != "AT"sv && cat != "AV"sv && cat != "BV"sv && cat != "WT"sv;
+		case CELL_GAME_GAMETYPE_HDD: return cat != "HG"sv && cat != "HM"sv && cat != "AM"sv && cat != "AP"sv && cat != "AT"sv && cat != "AV"sv && cat != "BV"sv && cat != "WT"sv;
```

Comment:
Why complicate it even further? simply != DG != GD && !.empty()

Reply:
Because there could be garbage in the string obviously

Reply:
May add such "garbage" checks in SFO loader itself if we are so anxious about it.